src/biokbase/narrative/tests/job_test_constants.py

```diff
@@ -1,16 +1,53 @@
 from .util import ConfigTests
 import copy
 
-CLIENTS = "biokbase.narrative.clients.get"
 
 config = ConfigTests()
-TEST_JOBS = config.load_json_file(config.get("jobs", "ee2_job_info_file"))
+TEST_JOBS = config.load_json_file(config.get("jobs", "ee2_job_test_data_file"))
+
+
+def get_retried_jobs(all_jobs):
+    # collect retried jobs
+    retried_jobs = {}
+    for job in all_jobs.values():
+        # save the first retry ID with the retried job
+        if "retry_ids" in job and len(job["retry_ids"]) > 0:
+            retried_jobs[job["job_id"]] = job["retry_ids"][0]
+    return retried_jobs
+
+
+def generate_error(job_id, err_type):
```

Comment:
The BE tests deal with various different error messages; these are all valid messages that may get passed to the frontend as part of a result set.